src/docs/product/performance/web-vitals.mdx

```diff
@@ -1,32 +1,30 @@
 ---
 
-[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and layout shift. Each data point provides insights about the overall <SandboxLink scenario="performance" projectSlug="react">performance</SandboxLink> of your application.
+[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and layout shift. Each data point provides insights about the overall [performance](/product/performance/) of your application.
 
-The in-browser Sentry SDKs collects <SandboxLink scenario="oneWebVitals" projectSlug="react">Web Vitals</SandboxLink> information (where supported) and adds that information to frontend <SandboxLink scenario="oneTransaction" projectSlug="react">transactions</SandboxLink>. These vitals are then summarized in several graphs for a quick overview of how each frontend transaction is performing for your users.
-
-You can visit the [Web Vitals](#web-vitals-page) page to get an overview of how page loads are performing in your application.
+The in-browser Sentry SDKs collects web vitals information (where supported) and adds that information to frontend [transactions](/product/performance/transaction-summary/). These web vitals are then summarized in several graphs for a quick overview of how each frontend transaction is performing for your users.
 
 ![Visualization of Web Vitals](./diagram-transaction-vitals.png)
 
 ## Core Web Vitals
 
-These <SandboxLink scenario="oneWebVitals" projectSlug="react">Web Vitals</SandboxLink> are considered the most important by Google and directly measure the user experience. [Google reports that as of May 2021, these metrics also impact your search ranking](https://developers.google.com/search/blog/2020/11/timing-for-page-experience).
+Core Web Vitals are called this because they're considered to be the most important by Google and directly measure the user experience. According to a May, 2021 [Google blog post](https://developers.google.com/search/blog/2020/11/timing-for-page-experience), these metrics also impact your search ranking.
 
 ### Largest Contentful Paint (LCP)
 
-[Largest Contentful Paint (LCP)](https://web.dev/lcp/) measures the render time for the largest content to appear in the viewport. This may be in any form from the document object model (DOM), such as images, SVGs, or text blocks. It is the largest pixel area in the viewport, thus most visually defining. LCP helps developers understand how long it takes before the user sees the main content on the page.
+[Largest Contentful Paint, (LCP)](https://web.dev/lcp/) measures how long it takes for the content that covers the largest pixel area in the viewport to render - in other words, how long before a user sees the main content on a page. This content may take any form from the document object model (DOM), such as an image, SVG, or text block.
```

Comment:
```suggestion
[Largest Contentful Paint (LCP)](https://web.dev/lcp/) measures how long it takes for the content that covers the largest pixel area in the viewport to render - in other words, how long before a user sees the main content on a page. This content may take any form from the document object model (DOM), such as an image, SVG, or text block.
```
I don't think you need the extra comma there. They're not used in the google dev docs either (https://web.dev/articles/lcp)